css/www/components/page.css

```diff
@@ -35,7 +35,7 @@
 }
 
 .www-component-anchors > li {
-  height: var(--navbar-height);
+  margin-bottom: var(--space-3);
 }
```

Comment:
fixed it in a PR , made it use MB3 classes on the li's HTML - should be updated in master now